r1cs/gadgets/crypto/Cargo.toml

```diff
@@ -57,4 +57,4 @@ llvm_asm = ["algebra/llvm_asm"]
 rand_xorshift = { version = "0.3.0" }
 algebra = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "audit_fixes", features = ["bls12_377", "bls12_381", "sw6", "bn_382"] }
 r1cs-std = { path = "../std", features = ["jubjub", "edwards_sw6", "bls12_377", "mnt4_753", "mnt6_753", "bn_382", "tweedle"] }
-r1cs-crypto = { path = "../crypto", features = ["mnt4_753", "mnt6_753", "bn_382", "tweedle"] }
\ No newline at end of file
+r1cs-crypto = { path = "../crypto", features = ["mnt4_753", "mnt6_753", "bn_382", "tweedle", "prf"] }
```

Comment:
Shouldn't be needed here

Reply:
I added "prf" to check the correctness of the build of the sha256 module. However now I removed it, and will add it only when calling the sha256 module from sc_audit_cryptolib. Correct?

Reply:
Correct